nose_ignores.txt

```diff
@@ -44,3 +44,4 @@
 --ignore-file=test_minimal_representation\.py
 --ignore-file=test_set_log_level\.py
 --ignore-file=test_field_parsing\.py
+--ignore-file=test_disks\.py
```

Comment:
Should also this add to yt/tests/test.yaml -- though tests ran just fine, so maybe we don't need to actually be doing that anymore?

Reply:
Indeed it seems that it's not needed here, but I don't know why TBH

Reply:
well I"ll hit approve and let you decide if you want to add it down in yt/test/test.yaml as well :)